ui/src/app/machines/views/MachineDetails/MachineStorage/AvailableStorageTable/AddLogicalVolume/AddLogicalVolume.tsx

```diff
@@ -107,7 +108,7 @@ export const AddLogicalVolume = ({
           mountOptions: "",
           mountPoint: "",
           name: initialName,
-          size: "",
+          size: formatBytes(disk.available_size, "GB").value,
```

Comment:
The second argument is actually supposed to be the unit of the first argument, which in this case is `"B"`, so this should be `formatBytes(disk.available_size, "B", { convertTo: "GB" }).value`.

Reply:
Done